### PR TITLE
provisioning: expand virtual media section

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: PROVISIONING.md [hunk 1]
```diff
@@ -30,23 +32,56 @@ Andy: Do we need to cover manual methods in our scope? Many, many methods...
...
 - Provisioning server maps Virtual Media with ISO image to provision, see https://github.com/openbmc/docs/blob/master/designs/virtual-media.md
-- Missing step : what script to call to start ? like Anaconda ?
+```
+POST https://<bmc_ip_address>/redfish/v1/Managers/bmc/VirtualMedia/CD/Actions/VirtualMedia.InsertMedia
+{
+"Image": "http://<web_server>/<image_name>.iso"
+}
+```
+- Call script to start installation from mounted image
+  - Question: how do we know what is the script name ? probably different for each Image...
```
[the remark this answers is not in the document]
Comment (mestery) [reply PROVISIONING.md:75]: Are you thinking of using systemd for this?

File: PROVISIONING.md [hunk 2]
```diff
@@ -30,23 +32,56 @@ Andy: Do we need to cover manual methods in our scope? Many, many methods...
...
 - Provisioning server causes reboot after or xPU reboots itself
+```
+POST https://<bmc_ip_address>/redfish/v1/Systems/1/Actions/ComputerSystem.Reset
```
[the remark this answers is not in the document]
Comment (mestery) [reply PROVISIONING.md:78]: What does "at this level" mean, do you mean the run level?

**Review by mestery — APPROVED** [review]

File: PROVISIONING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (glimchb) [reply PROVISIONING.md:75]: I mean how we provide the `kickstart` file

File: PROVISIONING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (glimchb) [reply PROVISIONING.md:78]: @avishayt yes, either the DPU reboots itself using `reboot` or we issue reboot externally via redfish. wanted to flush 2 options and discuss them

**Comment by glimchb** [comment]
@avishayt does this section accurately describes how RH using provisioning in OpenShift Installer Assist via Virtual Media today ? 
https://github.com/opiproject/opi-prov-life/blob/main/PROVISIONING.md#usbvirtual-media-provisioning

File: PROVISIONING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (avishayt) [reply PROVISIONING.md:75]: We embed into the ISO

File: PROVISIONING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (avishayt) [reply PROVISIONING.md:78]: OK

**Comment by avishayt** [comment]
> @avishayt does this section accurately describes how RH using provisioning in OpenShift Installer Assist via Virtual Media today ? https://github.com/opiproject/opi-prov-life/blob/main/PROVISIONING.md#usbvirtual-media-provisioning

Looks OK

File: PROVISIONING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (glimchb) [reply PROVISIONING.md:75]: does the ks have fixed name?

[the remark this answers is not in the document]
Comment (avishayt) [reply PROVISIONING.md:75]: We call it "ignition config" in coreOS.  It's placed in a standard place where ignition knows to find it.  Ignition runs it and can start any services we need, write files, etc.

[the remark this answers is not in the document]
Comment (glimchb) [reply PROVISIONING.md:75]: cool, clear now @avishayt thanks !

**Pull request merged**